src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -401,7 +406,7 @@ TEST_F(SQLPTRepresentationTest,
   // Check PT structure destroyed and tables number is 0
   query.Prepare(query_select);
   query.Next();
-  ASSERT_EQ(25, query.GetInteger(0));
+  ASSERT_EQ(kPolicyTablesNumber, query.GetInteger(0));
```

Comment:
@dev-gh I can see that you asser reading this constant. But I can't see that you insert it somewhere.